osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -369,7 +369,7 @@ private void handleEvent(string line)
                     // Some very old beatmaps had incorrect type specifications for their backgrounds (ie. using 1 for VIDEO
                     // instead of 0 for BACKGROUND). To handle this gracefully, check the file extension against known supported
                     // video extensions and handle similar to a background if it doesn't match.
-                    if (!OsuGameBase.VIDEO_EXTENSIONS.Contains(Path.GetExtension(filename)))
+                    if (!OsuGameBase.VIDEO_EXTENSIONS.Contains(Path.GetExtension(filename).ToUpperInvariant()))
```

Comment:
We're using `ToLower` elsewhere, so please stick to using this.

https://github.com/ppy/osu/blob/b8904fe7474e17bf48a7dbbbd60ae338dd77529c/osu.Game/OsuGameBase_Importing.cs#L35

Reply:
Done, switched to lowercase now.